config/notebook_config.py

```diff
@@ -0,0 +1,56 @@
+'''
+    Import the needed functionality
+    - tkinter : 
+        Python GUI library
+    - configuration.ProjectConfiguration
+        Configuration object that reads/writes to the configuration settings YAML file.
+    - configurationui.SettingsUpdate
+        tkinter based UI that dynamically loads any appropriate configuration file
+        and displays it to the user to alter the settings.
+'''
+from tkinter import * 
+from configuration import ProjectConfiguration
+from configurationui import SettingsUpdate
+
+project_configuration_file = "./project.yml"
+
+def configure_settings(configuration_yaml = None):
```

Comment:
in addition to parameter type declarations, python has optional return type declarations that I like to use. 
) -> ProjectConfiguration:

Reply:
Done